docs/sources/vendor/a10networks/vthunder.md

```diff
@@ -0,0 +1,33 @@
+# a10networks vthunder
+
+## Key facts
+
+* MSG Format based filter
+* Legacy BSD Format default port 514
+
+## Links
+
+| Ref                                   | Link                                                                             |
+|---------------------------------------|----------------------------------------------------------------------------------|
+| A10 Networks SSL Insight App          | <https://splunkbase.splunk.com/app/3937>                                         |
+| A10 Networks Application Firewall App | <https://splunkbase.splunk.com/app/3920>                                         |
+| A10 Networks L4 Firewall App          | <https://splunkbase.splunk.com/app/3910>                                         |
+
+
+## Sourcetypes
+
+| sourcetype               | notes                                                                                         |
+|--------------------------|-----------------------------------------------------------------------------------------------|
+| a10networks:vThunder:cef | None                                                                                          |
+
+## Source
+
+| source               | notes                                                                                             |
+|----------------------|---------------------------------------------------------------------------------------------------|
+| a10networks:vThunder | None                                                                                              |
+
+### Index Configuration
+
+| key        | source              | index          | notes          |
+|------------|---------------------|----------------|----------------|
+|a10_vthunder| a10networks:vThunder| netwaf         | none           |
```

Comment:
I failed to redirect event to another index with `splunk_metadata.csv` using this key:
```
a10_vthunder,index,main
```

Worked when changed to:
```
a10networks_vThunder,index,main
```